Problem with ESP32

I'm trying to complile the http demo in my ESP32 Dev kit and i'm reciving this error:

In file included from C:\Users\david\Documents\Arduino\libraries\ESP8266Audio-master\src\driver\SinglePinI2SDriver.cpp:34:0:

C:\Users\david\Documents\Arduino\libraries\ESP8266Audio-master\src\driver\SinglePinI2SDriver.h:24:2: error: #error "This driver is ESP8266 specific, only include it on ESP8266"

 #error "This driver is ESP8266 specific, only include it on ESP8266"

  ^

In file included from C:\Users\david\Documents\Arduino\libraries\ESP8266Audio-master\src\driver\SinglePinI2SDriver.cpp:34:0:

C:\Users\david\Documents\Arduino\libraries\ESP8266Audio-master\src\driver\SinglePinI2SDriver.h:28:19: fatal error: osapi.h: No such file or directory

compilation terminated.

exit status 1
Error compilando para la tarjeta ESP32 Dev Module.

--- a/src/driver/SinglePinI2SDriver.cpp
+++ b/src/driver/SinglePinI2SDriver.cpp
@@ -31,6 +31,8 @@
   Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
 */
 
+#if defined(ESP8266)
+
 #include "SinglePinI2SDriver.h"
 
 SinglePinI2SDriver::~SinglePinI2SDriver()
@@ -277,3 +279,5 @@ int SinglePinI2SDriver::getUnderflowCount()
 
 // Global instance
 SinglePinI2SDriver I2SDriver;
+
+#endif // defined(ESP8266)

--- a/src/driver/SinglePinI2SDriver.h
+++ b/src/driver/SinglePinI2SDriver.h
@@ -20,9 +20,7 @@
 */
 
 #pragma once
-#if !defined(ESP8266)
-#error "This driver is ESP8266 specific, only include it on ESP8266"
-#endif
+#if defined(ESP8266)
 
 #include <Arduino.h>
 #include "osapi.h"
@@ -83,3 +81,5 @@ class SinglePinI2SDriver
 
 // Global instance
 extern SinglePinI2SDriver I2SDriver;
+
+#endif // defined(ESP8266)

